scripts/error-codes/codes.json

```diff
@@ -1,252 +1,252 @@
 {
-  "0": "React.addons.createFragment(...): Encountered an invalid child; DOM elements are not valid children of React components.",
```

Comment:
This file is autogenerated. No need to change it.

Reply:
Sorry! I've forgotten to reverse this :cry: